gensim/models/tfidfmodel.py

```diff
@@ -49,8 +52,8 @@ class TfidfModel(interfaces.TransformationABC):
     Model persistency is achieved via its load/save methods.
     """
 
-    def __init__(self, corpus=None, id2word=None, dictionary=None,
-                 wlocal=utils.identity, wglobal=df2idf, normalize=True):
+    def __init__(self, corpus=None, id2word=None, dictionary=None, smartirs="ntc",
+                 wlocal=None, wglobal=None, normalize=None):
```

Comment:
Better to support backward compatibility, why you change default values?

Good solution - by default support default behavior and `smartirs=None`, but if user set `smartirs` - use it (and ignore wlocal, wglobal, ...)

Reply:
* It is backward compatible. Please check the smartirs value. :)
* This can be followed but I refrained because I removed `df2idf` function totally because it is redundant. Are you sure I should add it back?